src/main/RPBuilder.ts

```diff
@@ -35,9 +38,21 @@ export default class RPBuilder {
   claims?: ClaimOpts;
 
   // claims?: ClaimPayload;
+  vpFormatsSupported: Format;
+  idTokenSigningAlgValuesSupported: SigningAlgo[];
```

Comment:
Are all of these required parameters properties?

Reply:
yes and made a comment about required ones in the `SIOP.types`